src/test/ui/feature-gates/feature-gate-rustdoc_internals.rs

```diff
@@ -0,0 +1,5 @@
+#[doc(keyword = "match")] //~ ERROR: `#[doc(keyword)]` is meant for internal use only
+/// wonderful
+mod foo{}
```

Comment:
nit (feel free to ignore):

```suggestion
mod foo {}
```